<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="insertRowAction">
         <property name="text">
          <string>+строка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ConnectTo">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insertColumnAction">
         <property name="text">
          <string>+столбец</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeRowAction">
         <property name="text">
          <string>-строка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeColumnAction">
         <property name="text">
          <string>-столбец</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insertChildAction">
         <property name="text">
          <string>+узел</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Отправить файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="install">
         <property name="text">
          <string>Загрузить файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
